packages/mdc-drawer/README.md

```diff
@@ -234,6 +251,35 @@ topAppBar.listen('MDCTopAppBar:nav', () => {
 });
 ```
 
+### Modal Drawer
+
+Modal drawers are elevated above most of the app’s UI and don’t affect the screen’s layout grid.
+
+```html
+<body>
+  <header class="mdc-drawer mdc-drawer--modal">
```

Comment:
The root element is an `aside` in the demo page, but it's a `header` here?

There's also a case of this under Dismissible, and also a case of an outer `nav` (when the list inside content is also a `nav`).